sig-node/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing on SIG Node
+
+Welcome!
+
+## For Kubernetes Contributions
+
+https://github.com/kubernetes/community/tree/master/contributors/guide#contributor-guide
+
+### For Enhancements 
+
+https://github.com/kubernetes/enhancements/tree/master/keps/sig-node
+
+#### Helpful Links for Sig-Node
+
+Code:
+https://github.com/kubernetes/kubernetes/tree/master/cmd/kubelet
+https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet
+
+Development Resources:
+https://github.com/kubernetes/community/tree/master/contributors/devel#table-of-contents
+
+Shared space / Sub projects:
+https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node/
+https://github.com/kubernetes/community/tree/master/sig-node#subprojects
+
+Triage:
+https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/triage.md 
+
+
+## Getting Started
+
+    Task #1 : Compile kubelet
```

Comment:
is it node specific?

Reply:
Yes, this is Node's onboarding guide as opposed to project-wide.